File: Android/ex1115_calculate/app/src/main/java/com/example/hongseongbo/ex1115/MainActivity.java
package com.example.hongseongbo.ex1115;

import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void add(View v) {

        EditText num1 = (EditText) findViewById(R.id.et1);
        EditText num2 = (EditText) findViewById(R.id.et2);

        TextView resu = (TextView) findViewById(R.id.result);

        try {

            int n1 = Integer.parseInt(num1.getText().toString());
            int n2 = Integer.parseInt(num2.getText().toString());
            resu.setText(Integer.toString(n1 + n2));

        }catch(NumberFormatException e)
        {
            resu.setText("숫자가 정상적으로 입력되지 않았습니다.");
        }

    }

    public void sub(View v) {

        EditText num1 = (EditText) findViewById(R.id.et1);
        EditText num2 = (EditText) findViewById(R.id.et2);

        TextView resu = (TextView) findViewById(R.id.result);

        try {

            int n1 = Integer.parseInt(num1.getText().toString());
            int n2 = Integer.parseInt(num2.getText().toString());
            resu.setText(Integer.toString(n1 - n2));

        }catch(NumberFormatException e)
        {
            resu.setText("숫자가 정상적으로 입력되지 않았습니다.");
        }


    }

    public void mul(View v) {

        EditText num1 = (EditText) findViewById(R.id.et1);
        EditText num2 = (EditText) findViewById(R.id.et2);

        TextView resu = (TextView) findViewById(R.id.result);

        try {

            int n1 = Integer.parseInt(num1.getText().toString());
            int n2 = Integer.parseInt(num2.getText().toString());
            resu.setText(Integer.toString(n1 * n2));

        }catch(NumberFormatException e)
        {
            resu.setText("숫자가 정상적으로 입력되지 않았습니다.");
        }


    }

    public void div(View v) {

        EditText num1 = (EditText) findViewById(R.id.et1);
        EditText num2 = (EditText) findViewById(R.id.et2);

        TextView resu = (TextView) findViewById(R.id.result);

        try {

            int n1 = Integer.parseInt(num1.getText().toString());
            int n2 = Integer.parseInt(num2.getText().toString());
            resu.setText(Integer.toString(n1 / n2));

        }catch(NumberFormatException e)
        {
            resu.setText("숫자가 정상적으로 입력되지 않았습니다.");
        }


    }

}

File: Android/NongExam06_Spinner/app/src/main/java/com/example/nongexam6/MainActivity.kt
package com.example.nongexam6

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Adapter
import android.widget.AdapterView
import android.widget.ArrayAdapter
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {


    var nameArray = arrayOf("홍길동","김이박","갑을병정")

    var adapter: ArrayAdapter<String>? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        greetText.setOnClickListener(object:View.OnClickListener{
            override fun onClick(v: View?) {
                greetText.setText("문구를 클릭하였습니다.")
            }

        })

        adapter = ArrayAdapter(this,android.R.layout.simple_spinner_dropdown_item,nameArray)

        nameOption.adapter = adapter

        nameOption.onItemSelectedListener= object:AdapterView.OnItemSelectedListener{
            override fun onItemSelected(
                parent: AdapterView<*>?,
                view: View?,
                position: Int,
                id: Long
            ) {
                greetText.setText("${nameArray[position]}님 반갑습니다!")
            }

            override fun onNothingSelected(parent: AdapterView<*>?) {
                TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
            }

        }

    }
}
